lib/sprite.dart

```diff
@@ -1,20 +1,21 @@
 import 'dart:ui';
 
+import 'extensions/offset.dart';
 import 'extensions/vector2.dart';
 import 'palette.dart';
 
 class Sprite {
   Paint paint = BasicPalette.white.paint;
   Image image;
-  Rect src;
+  Rect bounds;
```

Comment:
I actually think `src` is clearer. This is to distinguish between the target rect (which is where in the screen it will be drawn). The `src` is where from the texture it's extracted.